.node-version

```diff
@@ -0,0 +1 @@
+14.19.0
```

Comment:
@baseballyama was this needed to get the tests to run locally?

Reply:
Oops sorry!

Reply:
removed! https://github.com/sveltejs/svelte/pull/8230/commits/7c4bef28c062530f8fabd10bf023edd534a4bbbd